src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java

```diff
@@ -223,6 +225,23 @@ public FFmpegOutputBuilder setComplexVideoFilter(String filter) {
     return this;
   }
 
+  /**
+   * Allows to add metadata to output video. Which keys are possible depends on the used output
+   * video codec.
+   * 
+   * @param key Metadata key, e.g. "comment"
+   * @param value Value to set for key
+   * @return This instance
+   */
+  public FFmpegOutputBuilder addMetaTag(String key, String value) {
+    this.video_enabled = true;
```

Comment:
To be consistent with elsewhere, can we do a quick checkNotNull(key), checkNotNull(value)